### PR TITLE
Give error when GitHub Bearer Token is not created with correct scope

### PR DESCRIPTION
**Pull request by tomasnorre**
This is related to my problem in issue #4 , so I thought it would help to improve the error handling. 

I'm not sure if this is handle correctly. I don't know if your minicli have similar to Symfony console  `$output->writeln('<error>foo</error>');` So I throw a RuntimeException, but I'm up for suggestions :) 

Update: I found it (minicli output), so I added a suggestion to how it could be done, let me know what you think.

### CODE REVIEW

File: app/Service/GithubServiceProvider.php [hunk 1]
```diff
@@ -75,6 +79,9 @@ public function getSponsorsList(): array
 
         if ($response['code'] == 200) {
             $data = json_decode($response['body'], true);
+            if(true) {
```
Comment (erikaheidi) [inline app/Service/GithubServiceProvider.php]: I don't understand this if here!

Comment (tomasnorre) [reply app/Service/GithubServiceProvider.php]: Upsi.. It's a for testing that I forgot to remove. Will take care when back from vacation. 

I don't recall right now, but think I wanted to check on `$data` will update PR when back.

Comment (tomasnorre) [reply app/Service/GithubServiceProvider.php]: It was a faulty commit from testing. I have reverted that part.

**Comment by erikaheidi** [comment]
Thank you @tomasnorre ! 🎉

**Pull request merged**